code/jvm/src/main/kotlin/org/ionproject/codegarten/database/dto/Delivery.kt

```diff
@@ -0,0 +1,19 @@
+package org.ionproject.codegarten.database.dto
+
+import java.time.OffsetDateTime
+
+data class Delivery(
+    val did: Int,
+    val number: Int,
+    val tag: String,
+    val due_date: OffsetDateTime,
```

Comment:
Why are we using `snake_case` here?

Reply:
We're taking advantage of JDBI's auto mapping from the database into Kotlin data classes. Since the names of the properties in the database use snake case, we're also using it here.